docs/index.rst

```diff
@@ -3,50 +3,175 @@
    You can adapt this file completely to your liking, but it should at least
    contain the root `toctree` directive.
 
-==================
-pAPRika
-==================
 
-pAPRika is a python toolkit for setting up, running, and analyzing free-energy molecular dynamics simulations.
 
-========
+.. role:: green-font
+.. role:: red-font
+.. role:: ignore-width
 
-Index
------
+.. |tick|    replace:: :green-font:`✓`
+.. |cross|   replace:: :red-font:`✕`
+.. |delta|   unicode:: U+0394
+.. |ast|     replace:: :ignore-width:`*`
+.. |dast|    replace:: :ignore-width:`**`
+.. |br| raw:: html
+
+   <br />
+
+.. image :: _static/images/paprika.png
+   :width: 350px
+   :align: left
+
+|
+|
+|
+|
+|
+|
+
+*pAPRika is a python toolkit for setting up, running, and analyzing free-energy molecular dynamics simulations based on the attach-pull-release (APR) method.*
+
+===================
+
+******
+Theory
+******
+
+
+Binding free-energy
+===================
+
+todo: explain APR theory (add images/diagram and equations)
+
+.. math ::
+
+   \begin{eqnarray}
+      \Delta G^{\circ}_{b} = -(W_{attach} + W_{pull} + W_{release-conf} + W_{release-std})
```

Comment:
Try:
```
  \Delta G^{\circ}_{b} = -(W_\text{attach} + W_\text{pull} + W_\text{release-conf} + W_\text{release-std})
```
if it will work, or `\mathrm` if not.

Reply:
That works. I'll change all subscripts to `\text{}`